src/player.cpp

```diff
@@ -12119,51 +12119,25 @@ int player::encumb( body_part bp ) const
  */
 int player::encumb(body_part bp, double &layers, int &armorenc) const
 {
-    int ret = 0;
-    double layer[MAX_CLOTHING_LAYER] = { };
-    int level = 0;
-    bool is_wearing_active_power_armor = false;
-    for( auto &w : worn ) {
-        if( w.active && w.is_power_armor() ) {
-            is_wearing_active_power_armor = true;
-            break;
-        }
-    }
+    int layer[MAX_CLOTHING_LAYER] = { };
 
-    for( auto& w : worn ) {
-        if( !w.covers(bp) ) {
-            continue;
-        }
+    for (auto w = worn.begin(); w != worn.end(); ++w) {
+        if (! w->covers(bp)) continue;
```

Comment:
Standards:
Each `if` should have braces, even when it is followed only by a single instruction.